README.md

```diff
@@ -42,19 +42,30 @@ The app is extremley easy to use (and self explantory):
 6. That's it :)
 
 The whole process is very simple and automated, so you don't need to worry about anything else.\
-Setting up should take less then a minute, and syncing should take about 1 second per photo (due to Google's API rate limitiations of 60 requests per user per minute)
+Setting up should take less then a minute, and syncing should take about 1 second per photo (due to Google's API rate limitations of 60 requests per user per minute)
 
 ## How to Run Locally
 
 In order for the backend to function, it requires an OAuth client id and secret + an API key.\
 Since (for obvious reasons) this is a private app, you will need to create one for your own.\
 You can see instructions on how to do that [here](https://developers.google.com/workspace/guides/create-credentials).\
-Once you do that, create the file `server/.env`, and set the following environment variables:
+You will need OAuth Client ID credentials, as well as API key credentials.
 
-- `GOOGLE_CLIENT_ID`
-- `GOOGLE_CLIENT_SECRET`
+Rename [`server/sample.env`](server/sample.env) to `server/.env` and fill in the values.
 
-You also need to update the `CLIENT_ID` and `API_KEY` variables in [`web/src/pages/GoogleAuth.vue`](web/src/pages/GoogleAuth.vue).
+```env
+GOOGLE_CLIENT_ID=
```

Comment:
I know I previously asked why these vars have different names, but since these are 2 different pairs of ID/secret, it'd be better for the names to be differentiated, with the difference between them clear by the name.